<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MechSystemClass</class>
 <widget class="QMainWindow" name="MechSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MechSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>18</y>
      <width>322</width>
      <height>144</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="timeDisplay">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="heatBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="heatLabel">
           <property name="text">
            <string>Heat:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="heatDisplay">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>h</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="damageLabel">
           <property name="text">
            <string>Damage:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="damageDisplay">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>d</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="heatPercentLabel">
         <property name="text">
          <string>Heat %:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="heatPercentDisplay">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alphaButton">
         <property name="text">
          <string>Alpha</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Mech</string>
       </property>
       <widget class="QCheckBox" name="doubleHeatsinkCheck">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>211</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Double Heatsinks</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>91</width>
          <height>72</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="techSelectLayout">
         <item>
          <widget class="QRadioButton" name="selectSphereButton">
           <property name="text">
            <string>IS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">factionTech</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="selectClanButton">
           <property name="text">
            <string>Clan</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">factionTech</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>140</y>
          <width>118</width>
          <height>70</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="extHeatSinkLayout">
         <item>
          <widget class="QLabel" name="extHeatSinkLabel">
           <property name="text">
            <string>Ext. HS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="extHeatSinkInput">
           <property name="maximum">
            <number>80</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>73</width>
          <height>70</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="engineLayout">
         <item>
          <widget class="QLabel" name="engineLabel">
           <property name="text">
            <string>Engine:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="engineSelect">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="singleStep">
            <number>25</number>
           </property>
           <property name="value">
            <number>250</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>121</x>
          <y>141</y>
          <width>75</width>
          <height>68</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="intHeatSinkLayout">
         <item>
          <widget class="QLabel" name="intHeatSinkLabel">
           <property name="text">
            <string>Int. HS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="intHeatSinkDisplay">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>inths</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mechsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pauseButton</sender>
   <signal>pressed()</signal>
   <receiver>MechSystemClass</receiver>
   <slot>pauseSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>pressed()</signal>
   <receiver>MechSystemClass</receiver>
   <slot>stopSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>pressed()</signal>
   <receiver>MechSystemClass</receiver>
   <slot>startSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaButton</sender>
   <signal>pressed()</signal>
   <receiver>MechSystemClass</receiver>
   <slot>fireAlpha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startSimulation()</slot>
  <slot>resetSimulation()</slot>
  <slot>pauseSimulation()</slot>
  <slot>fireAlpha()</slot>
  <slot>stopSimulation()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="factionTech"/>
 </buttongroups>
</ui>
